posts/2023-08-22-ArgoCD-Configuration-Drift.adoc

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: "A Solution to Configuration Drift - Deploying Open Liberty with GitOps"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/danielguinan1
+author_github: https://github.com/danielguinan1
+seo-title: TITLE - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: "DESCRIPTION"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+additional_authors: 
+- name: Monica Tamboli
+  github: https://github.com/mtamboli
+  image: https://avatars0.githubusercontent.com/mtamboli
+---
+= A Solution to Configuration Drift - Deploying Open Liberty with GitOps 
+Daniel Guinan <https://github.com/danielguinan1>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// // // // // // // //
```

Comment:
Can we remove this commented section to enable clearer readability?

Reply:
https://github.com/OpenLiberty/blogs/blob/3bbc9670b8e2f23cd67e4ee7a48d05422e4004c8/posts/2023-08-22-ArgoCD-Configuration-Drift.adoc?plain=1#L23